app/components/file-viewer/audio-player.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class AudioPlayer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.player = null;
+
+    this.playAudio = this.playAudio.bind(this);
+
+    this.state = {
+      playing: false
+    };
+  }
+
+  componentDidMount() {
+    this.player.controlsList = "nodownload"; // Non-spec option for Chrome browsers to hide the display of a download button
+  }
+
+  componentDidUpdate() {
+  }
+
+  playAudio(playing) {
+    const { player } = this;
+    if (!player) return;
+
+    this.setState({ playing });
+
+    if (playing) {
+      player.play();
+    } else {
+      player.pause();
+    }
+  }
+
+  endAudio() {
+    this.setState({ playing: false });
+  }
+
+  render() {
+    // const rates = [0.25, 0.5, 1];
+    return (
+      <div className="subject-audio-frame">
+        <audio
+          className="subject"
+          controls={true}
+          ref={(element) => { this.player = element; }}
+          src={this.props.src}
+          type={`${this.props.type}/${this.props.format}`}
+          preload="auto"
```

Comment:
This is only a browser hint but should this be auto? Considering asset sizes it may be best to avoid automatic download of the media asset unless the user presses play. 

'metadata' might be a better fit, https://developer.mozilla.org/en/docs/Web/HTML/Element/audio